k8s-daemonset/helloworld/world.py

```diff
@@ -4,10 +4,11 @@
 app = Flask(__name__)
 
 pod_ip = os.getenv("POD_IP")
+target_world = os.getenv("TARGET_WORLD", "world")
```

Comment:
Totally agree -- we could put something in the helloworld/README.md file, referencing the continuous deployment walkthrough.  I'd like to do that as part of a follow up branch though.

As an aside, I think that the environment variable itself is not confusing, and I could imagine we would use it in a variety of different settings beyond just continuous deploy.  The orphaned text file, on the other hand, could use some explanation.